lua/orgmode/treesitter/headline.lua

```diff
@@ -475,6 +475,43 @@ function Headline:parse(pattern)
   return matching_nodes[1], match
 end
 
+function Headline:get_drawer(name)
+  local section = self.headline:parent()
+  local body = section:field('body')
+  if body and #body then
```

Comment:
I think something like this would be a bit more readable:
```suggestion
  if not body then
    return nil
  end
  body = body[1]
```

#body is always truthy in lua, no matter if it's 0 or not

Reply:
TIL, thanks